src/programs/aws.ts

```diff
@@ -186,7 +186,7 @@ export const AWSProgram = (opts: OptionValues) => {
           };
 
     const CONTAINER_SERVICE_URL = containerService.url.apply(url =>
-      url.endsWith('/') ? url.slice(0, -1) : url
+      url.endsWith('/') ? url.slice(0, -1) : url,
```

Comment:
While you're here, do you mind making `CONTAINER_SERVICE_URL` -> `containerServiceUrl`? I didn't want to bother a contributor with this nit, but I'm happy to bug you with it 😉

Reply:
Done in https://github.com/backstage/backstage-deploy/pull/33/commits/094203933370acec9c9a5281c5b7dc62ee635c60